openpype/lib/transcoding.py

```diff
@@ -1345,6 +1341,40 @@ def get_rescaled_command_arguments(
     return command_args
 
 
+def _get_image_dimensions(application, input_path, log):
+    """Uses 'ffprobe' first and then 'oiiotool' if available to get dim.
+
+    Args:
+        application (str): "oiiotool"|"ffmpeg"
+        input_path (str): path to image file
+        log (Optional[logging.Logger]): Logger used for logging.
+    Returns:
+        (tuple) (int, int, dict) - (height, width, sample_aspect_ratio)
+    Raises:
+        RuntimeError if image dimensions couldn't be parsed out.
+    """
+    # ffmpeg command
+    input_file_metadata = get_ffprobe_data(input_path, logger=log)
+    stream = input_file_metadata["streams"][0]
+    input_width = int(stream["width"])
+    input_height = int(stream["height"])
+
+    # fallback for weird files with width=0, height=0
+    if (input_width == 0 or input_height == 0) and application == "oiiotool":
+        # Load info about file from oiio tool
+        input_info = get_oiio_info_for_input(input_path, logger=log)
```

Comment:
Oiio is not always supported!

Reply:
I would say the issue is that it was always looking for first stream. There should be loop over all of them, use only those with `"video"` `"code_type"`.